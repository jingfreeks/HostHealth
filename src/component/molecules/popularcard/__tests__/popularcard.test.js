import React from 'react';
import {render} from '@testing-library/react-native';
import PopularCard from '../popularcard';

describe('Popular Card Component', () => {
  it('Should work as expected to get snapshot', () => {
    const all = render(
      <PopularCard
        item={{image: '', city: 'test city', matches: 'test', salary: '$100'}}
      />,
    );
    expect(all.toJSON()).toMatchSnapshot();
  });
});
