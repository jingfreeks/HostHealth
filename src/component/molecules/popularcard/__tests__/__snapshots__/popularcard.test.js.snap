// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Popular Card Component Should work as expected to get snapshot 1`] = `
<View
  style={
    [
      {
        "alignSelf": "stretch",
        "borderColor": "#E6E6E6",
        "borderRadius": 10,
        "borderWidth": 1,
        "flexBasis": 0,
        "flexDirection": "row",
        "flexGrow": 1,
        "flexShrink": 1,
        "marginTop": 20,
      },
    ]
  }
>
  <View
    style={
      [
        {
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
        },
      ]
    }
  >
    <Image
      resizeMode="stretch"
      source={
        {
          "uri": "",
        }
      }
      style={
        [
          {
            "borderBottomLeftRadius": 10,
            "borderTopLeftRadius": 10,
            "flexBasis": 0,
            "flexGrow": 1,
            "flexShrink": 1,
            "height": 100,
          },
        ]
      }
    />
  </View>
  <View
    style={
      [
        {
          "flexBasis": 0,
          "flexGrow": 3,
          "flexShrink": 1,
          "marginBottom": 5,
          "marginTop": 5,
        },
      ]
    }
  >
    <View
      style={
        [
          {
            "flexDirection": "row",
            "paddingBottom": 5,
            "paddingLeft": 5,
            "paddingRight": 5,
            "paddingTop": 5,
          },
        ]
      }
    >
      <Text
        style={
          [
            {
              "color": "#03606A",
              "fontFamily": "Avenir",
              "fontSize": 16,
              "fontWeight": "900",
            },
          ]
        }
      >
         
        <Text
          style={
            [
              {
                "color": "#0e3f6c",
                "fontFamily": "Avenir",
                "fontSize": 12,
                "fontWeight": "500",
              },
            ]
          }
        />
      </Text>
    </View>
    <View
      style={
        [
          {
            "flexDirection": "row",
            "paddingBottom": 5,
            "paddingLeft": 5,
            "paddingRight": 5,
            "paddingTop": 5,
          },
        ]
      }
    >
      <View
        style={
          [
            {
              "flexBasis": 0,
              "flexGrow": 1,
              "flexShrink": 1,
            },
          ]
        }
      >
        <Text
          style={
            [
              {
                "color": "#0e3f6c",
                "fontFamily": "Avenir",
                "fontSize": 14,
                "fontWeight": "900",
              },
            ]
          }
        >
          test
           
          <Text
            style={
              [
                {
                  "color": "#0e3f6c",
                  "fontFamily": "Avenir",
                  "fontSize": 12,
                  "fontWeight": "500",
                },
              ]
            }
          >
            Matches
          </Text>
        </Text>
      </View>
      <Text
        style={
          [
            {
              "color": "#000000",
              "fontFamily": "Avenir",
              "fontSize": 16,
              "fontWeight": "900",
            },
          ]
        }
      >
        $100
      </Text>
      <View
        style={
          [
            {
              "alignSelf": "flex-end",
              "flexDirection": "row",
            },
          ]
        }
      >
        <Text
          style={
            [
              {
                "color": "#000000",
                "fontFamily": "Avenir",
                "fontSize": 12,
                "fontWeight": "400",
              },
            ]
          }
        >
           /wk
        </Text>
      </View>
    </View>
  </View>
</View>
`;
